### DISCUSSION TITLE
Investigate Python.jl for diffeqpy

### DISCUSSION THREAD

**Discussion by ChrisRackauckas** [cjdoris/PythonCall.jl#12] [General]
Hey! Found this repo, looks cool! I'm looking into it as a possible new backing for https://github.com/SciML/diffeqpy . Specifically the issue is whether overloads in Julia for ModelingToolkit.jl could work well enough from Python. They do in diffeqr via JuliaCall, so it seems like it's just a wrapper issue with pyjulia. https://github.com/SciML/diffeqpy/issues/57#issuecomment-665052615 in particular highlights such an issue. https://github.com/SciML/diffeqpy/issues/78 is the same thing. If there's a good answer here, then I think got a new lead!

**Comment by cjdoris** [comment]
This is great to hear! Please let me know what works and what doesn't. By the way I renamed the Python package to `juliaaa` from `julia` to avoid the clash with the already existing package. I'm still looking for a good name.

Mild warning: Python.jl is still under development and changing quite fast, there are no promises about the API staying fixed at the moment (though I doubt there will be major changes). Version 0.1 isn't even registered yet, though I'm hoping to get to version 1.0 in quite short time.
